data/Templates/Hl7v2/Resource/_Observation.liquid

```diff
@@ -58,24 +58,6 @@
                 {% endif -%}{% endif -%}
             },
         },
-        "valueRange":
```

Comment:
Will this deleted section cause "parse errors"?

Reply:
Yes, or there will be a `{"While building a POCO: Element 'value' must not repeat (at Bundle.entry[5].resource[0].valueRange[0])"}` error.